backend/main.py

```
@@ -0,0 +1,372 @@
+    score: int
+
+# --- Persistence (JSON File) ---
+DATA_FILE = "data.json"
+
+def load_data():
+    if not os.path.exists(DATA_FILE):
+        return {"favorites": [], "history": []}
+    try:
+        with open(DATA_FILE, "r") as f:
+            return json.load(f)
+    except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)